docs/devel/go-code.md

```diff
@@ -53,6 +55,17 @@ and tips that our developers have found useful.
 - Consider making a separate Go tree for each project, which can make overlapping dependency management much easier. Remember to set the `$GOPATH` correctly! Consider [scripting](https://gist.github.com/timstclair/17ca792a20e0d83b06dddef7d77b1ea0) this.
 - Emacs users - setup [go-mode](https://github.com/dominikh/go-mode.el)
 
+## How to update the Go version
```

Comment:
Ah cool will update

On Tuesday, July 19, 2016, Brian Grant <notifications@github.com> wrote:

> In docs/devel/go-code.md
> <https://github.com/kubernetes/kubernetes/pull/29200#discussion_r71416499>
> :
>
> > @@ -53,6 +55,17 @@ and tips that our developers have found useful.
> >  - Consider making a separate Go tree for each project, which can make overlapping dependency management much easier. Remember to set the `$GOPATH` correctly! Consider [scripting](https://gist.github.com/timstclair/17ca792a20e0d83b06dddef7d77b1ea0) this.
> >  - Emacs users - setup [go-mode](https://github.com/dominikh/go-mode.el)
> >
> > +## How to update the Go version
>
> It would be useful to say something about when would someone need/want to
> update the Go version.
>
> This should also point to the place where we document the supported
> version:
>
>
> https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md#go-development-environment
>
> That's enforced somewhere. Is it in one of the places you reference?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29200/files/e3fddcc9df118b9e7afd4356bb3aeef0b4ab370b#r71416499>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABYNbJp1k7RMZg-ob6ZM-KwnA9IHd979ks5qXTMsgaJpZM4JPy4R>
> .
>